Automattic/pocket-casts-ios
# End of Year: Implement design notes for the listening time

| 📘 Project: #376 | 🛫 Depends on: #557 |
|:---:|:---:|

This implements the following notes:
- Always show the full time unit string
   - This changes `1 hour 2 min` to `1 hour 2 minutes`
- Adjusts the font sizes / weights to match the designs
- Adjusts the line heights
- Adjusts the spacing between the diagonal stack and the labels to be inline with the designs
- I also tried adjusting the centering of the stack for larger devices, but it wasn't working on so I skipped it. 

## Screenshots

### Design vs Implementation Comparison
<p align="center"><img src="https://user-images.githubusercontent.com/793774/205175467-9bd4a431-2d1a-445e-aa0d-bafdb10eb596.png" width="100%" /></p>

### Devices
| iPhone SE - 14.1 | iPhone 13 | iPhone 14 Pro Max |
|:---:|:---:|:---:|
|<img src="https://user-images.githubusercontent.com/793774/205175679-6e3b2e0d-06c5-4f6d-b316-9900c98d5983.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205175625-db311e62-c26b-4ff5-a312-cc65576be599.png" width="320" />|<img src="https://user-images.githubusercontent.com/793774/205175857-d76fe314-478c-4250-97ca-641fdc0246c1.png" width="320" />|

## To test

1. Launch the app
2. Tap on the Profile Tab > End of Year Card
3. Tap to the second screen
4. ✅ Verify the view looks like the designs and looks good on all screen sizes

## Checklist
